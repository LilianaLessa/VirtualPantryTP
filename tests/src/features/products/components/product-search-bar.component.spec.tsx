// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-nocheck
import * as React from "react";
import * as renderer from "react-test-renderer";
import ProductSearchBar from "../../../../../src/features/products/components/product-search-bar.component";

describe("App", () => {
  it("snapshot test", () => {
    const tree = renderer
      .create(<ProductSearchBar addItemToShoppingListCallback={() => {}} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
