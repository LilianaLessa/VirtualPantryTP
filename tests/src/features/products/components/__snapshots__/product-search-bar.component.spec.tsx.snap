// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View
  className="sc-beqWaB knyjdu"
>
  <View
    className="sc-gueYoa dcFCCb"
  >
    <Text />
    <TextInput
      className="sc-eDDNvR zPvDu"
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Search"
      value=""
    />
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        {
          "opacity": 1,
        }
      }
    >
      <Text />
    </View>
  </View>
</View>
`;
